config/initializers/mcommunity.rb

```diff
@@ -0,0 +1,43 @@
+require 'erb'
+require 'yaml'
+
+# Yeah, there's many better ways to do this, but later.
+def load_yml
+  cfg_file="#{::Rails.root}/config/mcommunity.yml"
+
+  begin
+    mcommunity_erb = ERB.new(IO.read(cfg_file)).result(binding)
+  rescue StandardError, SyntaxError => e
+    raise("#{cfg_file} could not be parsed with ERB. \n#{e.inspect}")
+  end
+
+  begin
+    mcommunity_yml = YAML::load(mcommunity_erb)
+  rescue => e
+    raise("#{cfg_file} was found, but could not be parsed.\n#{e.inspect}")
+  end
+
+  if mcommunity_yml.nil? || !mcommunity_yml.is_a?(Hash)
+    raise("#{cfg_file} was found, but was blank or malformed.\n")
+  end
+
+  begin
+    raise "The #{::Rails.env} environment settings were not found in #{cfg_file}" unless mcommunity_yml[::Rails.env]
+    mcommunity_cfg = mcommunity_yml[::Rails.env].symbolize_keys
+  end
+
+  mcommunity_cfg
+end
+
+
+yml_cfg = load_yml
+LdapConfig = Umrdr::LdapConfig.new
```

Comment:
This is not needed.  You're creating an instance of Umrdr::LdapConfig and assigning it to a variable that makes it look like a class.